### PR TITLE
chat: 24px sigils

### PR DESCRIPTION
**Pull request by jamesacklin**
Resizes chat sigils and avatars to 24px square and slightly respaces some elements to keep everything in line.

- Keeps sigils themselves to 16px square, but pads enough around them to appear as 24px boxes. This is to keep them aligned to the author's name.
- Message author name type size is now uniform to the message content.
- Sigil, author, and timestamp are padded over to line up with the channel title in the header.
- Subsequent (non-sigil'd) messages are padded to align with author name.

image

image

Keeping as a draft right now to iterate with design.

Fixes urbit/landscape#460

### CODE REVIEW

File: pkg/interface/src/views/apps/chat/components/ChatInput.tsx [hunk 1]
```diff
@@ -158,7 +170,7 @@ class ChatInput extends Component<ChatInputProps, ChatInputState> {
         className='cf'
         zIndex={0}
       >
-        <Row p='2' alignItems='center'>
+        <Row p='12px 8px 12px 12px' alignItems='center'>
```
Comment (matildepark) [inline pkg/interface/src/views/apps/chat/components/ChatInput.tsx:173]: isnt this just '3 2 3 3'? we should opt for indigo values before hard coding

Comment (jamesacklin) [reply pkg/interface/src/views/apps/chat/components/ChatInput.tsx:173]: Unfortunately not -- https://github.com/urbit/indigo-react/blob/1b93c5e502e28fa9e459f5edcb2bd428eeb82585/src/themes/light.ts#L150

Comment (matildepark) [reply pkg/interface/src/views/apps/chat/components/ChatInput.tsx:173]: So our mocks are divergent from Indigo's spacing? Seems weird

**Review by urcades — APPROVED** [review]
I Truly Love This — We should "Ship It" asap

**Review by matildepark — APPROVED** [review]
The code is straightforward and it doesn't break other renders of sigils, works on mobile, works on desktop. Seems fine to me

File: pkg/interface/src/views/apps/chat/components/ChatInput.tsx (hunk 1 above)
Comment (urcades) [reply pkg/interface/src/views/apps/chat/components/ChatInput.tsx:173]: The mocks are derived from gav's actual indigo chat components, with the exception of the left-hand margin, which is a bit larger for ensuring highlighting will work sanely when that ships, and that timestamps fit in without being too squashed

Comment (matildepark) [reply pkg/interface/src/views/apps/chat/components/ChatInput.tsx:173]: The issue here is that 12px doesn't appear to be an indigo spacing value

**Pull request merged**